pkg/cmd/attestation/verify/policy_test.go

```diff
@@ -56,6 +56,29 @@ func TestNewEnforcementCriteria(t *testing.T) {
 		require.Equal(t, "(?i)^https://github/foo", c.SANRegex)
 	})
 
+	t.Run("sets SANRegex using opts.Repo", func(t *testing.T) {
+		opts := &Options{
+			ArtifactPath: artifactPath,
+			Owner:        "foo",
```

Comment:
Sorry, I should've given you this review earlier but I didn't have time to read thru the tests.

First up, this test should not have the Owner set, yeah?

```suggestion
```

and might as well test SourceRepositoryURI and SourceRepositoryOwnerURI while you're at it.

Secondly,

- I feel like we should at least lightly exercise the mutually exclusive options. Like, have one test case that sets Repo and SAN, and checks that that the Repo value is ignored – but set in the sourcerepo uri.
- do we test out the tenant anywhere?

Reply:
Added some more tests to cover these cases